src/parameters/parameters_ode_solver.cpp

```diff
@@ -102,7 +104,9 @@ void ODESolverParam::declare_parameters (dealii::ParameterHandler &prm)
                           " euler_ex | "
                           " euler_im | "
                           " dirk_2_im | "
-                          " dirk_3_im"),
+                          " dirk_3_im | "
+                          " RK3_2_5F_3SStarPlus | "
+                          " RK4_3_5_3SStar "),
                           "Runge-kutta method to use. Methods with _ex are explicit, and with _im are implicit."
```

Comment:
Maybe add another line to this description indicating that 2SStarPlus and 3SStar are low-storage RK methods.